simqso/data/addfilters.py

```diff
@@ -19,3 +19,17 @@ def add_speclite_filters(speclitepath):
             filtDat.append(fits.BinTableHDU(ftab,name=name+'-'+b))
     filtDat.close()
 
+def add_mzls_bass_filters(speclitepath):
```

Comment:
It would be better to do this instead of copy-paste:
`def add_speclite_filters(speclitepath, toadd):`
then
```
def add_mzls_bass_filters(speclitepath):
    add_speclite_filters(speclitepath, [ ('BASS-MzLS','BASS','gr'), ('BASS-MzLS','MzLS','z') ])
```

Sorry it's nitpicky, but if you don't mind making that quick change then I'll merge. Thanks for the contribution/bugfixes!

Reply:
Done!